packages/components/organisms/f-checkout/src/store/tests/checkout.module.test.js

```diff
@@ -636,6 +645,27 @@ describe('CheckoutModule', () => {
                             expect(commit).toHaveBeenCalledWith(UPDATE_ERRORS, []);
                         }
                     });
+
+                    it('should dispatch `updateMessage` with checkoutIssue', async () => {
+                        // Arrange
+                        // eslint-disable-next-line prefer-promise-reject-errors
+                        axios.post = jest.fn(() => Promise.reject({
+                            status: 400,
+                            response: {
+                                data: {
+                                    errorCode: 'DuplicateOrder'
+                                }
+                            }
+                        }));
+
+                        try {
+                            // Act
+                            await placeOrder({ commit, state, dispatch }, payload);
+                        } catch {
+                            // Assert
+                            expect(dispatch).toHaveBeenCalledWith('updateMessage', message);
```

Comment:
Given that this might not be called if the placeOrder function erroneously returns without throwing, it would be a good idea to have a [`expect.hasAssertions()`](https://jestjs.io/docs/expect#expecthasassertions) statement earlier in the test, to ensure that we are actually making this assertion.